samza-core/src/main/java/org/apache/samza/runtime/LocalContainerRunner.java

```diff
@@ -72,6 +72,7 @@ public void runTask() {
 
   @Override
   public void run(StreamApplication streamApp) {
+    super.run(streamApp);
```

Comment:
Kill is never called here?

Reply:
Yeah I think so. `LocalContainerRunner.kill(...)` will throw `UnsupportedOperationException`.

Reply:
Are there any issues with leaving the SystemAdmins running in this case?

Reply:
I think there is no issue with leaving SystemAdmins running. It seems reasonable SystemAdmins to keep running because `streamManager.getStreamPartitionCounts()` can potentially be run anytime when the LocalContainerRunner is running. What do you think?